executor/batch_point_get.go

```diff
@@ -210,7 +214,11 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 				continue
 			}
 
-			physID := getPhysID(e.tblInfo, idxVals[e.partPos].GetInt64())
+			physID, err := getPhysID(e.tblInfo, e.partExpr, idxVals[e.partPos].GetInt64())
+			if err != nil {
+				continue
```

Comment:
As for `BathGet`, if one data point in the where clause `where col in (....)` doesn't match any partition, we can just safely ignore it, and continue trying to fetch the next data point.

TableDual is not applicable here because we have already build out executor in `buildBatchPointGet` instead of `buildTableDual`.